sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/MarkAssemblyWithAssemblyVersionAttribute.cs

```diff
@@ -31,7 +31,7 @@ namespace SonarAnalyzer.Rules.CSharp
     public sealed class MarkAssemblyWithAssemblyVersionAttribute : MarkAssemblyWithAttributeBase
```

Comment:
I've noticed that there is also `MarkAssemblyWithComVisibleAttribute` and `MarkAssemblyWithClsCompliantAttribute` which refer to the assembly with this "Mark **this** assembly with ".

What to you think about updating also the remaining rules?

Reply:
I thought about it, but wasn't sure whether to do it in this PR or not. But I guess it makes sense. I updated PR to also update these rules message